DegenPrimer/UnifiedNN.py

```diff
@@ -99,7 +99,7 @@ class UnifiedNN(object):
     _inited = False
     
     #paths to the thermodynamic tables
-    _possible_paths = ('./', '/usr/local/share/degen_primer/', '/usr/share/degen_primer/')
+    _possible_paths = ('./', '/usr/local/share/degen_primer/', '/usr/share/degen_primer/', '/home/npilshchikova/DegenPrimer/data')
```

Comment:
You shouldn't add your personal paths to the common distribution.
Why not install DegenPrimer the way it is meant to be installed?
Or add the local installation path here: something like `os.path.expanduser('~/.local/share/degen_primer')`.
Just check what it actually is when you install into a user folder.